docs/reference/launch-configuration.md

```diff
@@ -36,13 +37,23 @@ The file may be called by any name but should have the following structure:
         }
       ]
     }
-  ]
+  ],
+  "streamingQueryInterval": 4,
+  "runtimeFlags": [
+    {
+      "name": "...",
+      "value": "..."
+    }
+  ]   
 }
 ```
 
 | Field | Required/Optional | Description | 
 | :------------- | :------------- | :------- |
+| `template` | Required | Defines the compute resources your deployment needs (i.e., its ‘topology’). See [deployment templates](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#templates) for more details. |
 | `dimensionsInWorldUnits` | Required | Describes how big the world should be. "x" and "z" parameters must both be greater than 0. |
 | `loadBalancing` | Required | The new load balancing configuration as documented in [load balancing with layers](https://docs.improbable.io/reference/latest/shared/worker-configuration/load-balancing). |
 | `snapshot` | Optional | Specifies which snapshot file to start the deployment from and how often to take a snapshot while the deployment is running. <br><br> It contains two optional fields: `takeSnapshotIntervalSeconds` (optional) and `startDeploymentFromSnapshotFile` (optional). |
-| `workerFlags` | Optional | Specifies any additional flags to pass to workers. Each element contains two fields: `workerType` (required) and `flags` (required). |
\ No newline at end of file
+| `workerFlags` | Optional | Specifies any additional flags to pass to workers. Each element contains two fields: `workerType` (required) and `flags` (required). |
+| `streamingQueryInterval` | Optional | Period in seconds between successive streaming query updates. The default value is `4` seconds. |
+| `runtimeFlags` | Optional | The runtime flags that can control advanced runtime features. Equivalent to [legacy flags](https://docs.improbable.io/reference/latest/shared/reference/file-formats/launch-config#legacy-flags).|
```

Comment:
How are you using "runtime" here? If you're talking about "the SpatialOS Runtime", it needs a capital; otherwise, it's fine as it is.

Reply:
Yes, changed it to "Runtime".